base_layer/wallet/src/connectivity_service/initializer.rs

```diff
@@ -36,13 +36,15 @@ use crate::client::http_client_factory::HttpClientFactory;
 
 pub struct WalletConnectivityInitializer<TClientFactory: HttpClientFactory> {
     http_node_url: Url,
+    https_seed_url: Url,
     phantom: std::marker::PhantomData<TClientFactory>,
 }
 
 impl<T: HttpClientFactory> WalletConnectivityInitializer<T> {
-    pub fn new(http_node_url: Url) -> Self {
+    pub fn new(http_node_url: Url, https_seed_url: Url) -> Self {
```

Comment:
Elsewhere in the code, we talk about `http`, here it is called `https`. This should be consistent.